<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>630</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>571</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabSanityCheck">
    <attribute name="title">
     <string>Sanity Check</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>586</width>
       <height>491</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="rowCount">
         <number>2</number>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>200</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <row/>
        <row/>
        <column/>
        <column/>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="retestButton">
     <property name="geometry">
      <rect>
       <x>465</x>
       <y>515</y>
       <width>125</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Re-run tests</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passFailLineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>515</y>
       <width>286</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="autoButton">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>515</y>
       <width>125</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply auto-fixes</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabPublish">
    <attribute name="title">
     <string>Publish</string>
    </attribute>
    <widget class="QTableWidget" name="publishTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>205</y>
       <width>586</width>
       <height>141</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="rowCount">
      <number>2</number>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>200</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <column/>
     <column/>
     <column/>
    </widget>
    <widget class="QPushButton" name="publishButton">
     <property name="geometry">
      <rect>
       <x>445</x>
       <y>515</y>
       <width>150</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Publish</string>
     </property>
    </widget>
    <widget class="QPushButton" name="publishOpenFolder">
     <property name="geometry">
      <rect>
       <x>233</x>
       <y>515</y>
       <width>150</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Publish Folder</string>
     </property>
    </widget>
    <widget class="QPushButton" name="rescanButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>515</y>
       <width>150</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Use this to rebuild the Asset Manger's JSON file if it becomes lost. FOR EMERGENCIES ONLY.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(35, 90, 150);</string>
     </property>
     <property name="text">
      <string>Rebuild JSON File</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="publishLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>355</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>&lt;filename&gt;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="commentTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>405</y>
       <width>586</width>
       <height>71</height>
      </rect>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>385</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Comment:</string>
     </property>
    </widget>
    <widget class="QLabel" name="publishPathLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>485</y>
       <width>581</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Output:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="relistButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>355</y>
       <width>150</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove from Blacklist</string>
     </property>
    </widget>
    <widget class="QPushButton" name="blacklistButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>445</x>
       <y>355</y>
       <width>150</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Add to Blacklist</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>586</width>
       <height>161</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QComboBox" name="filterStatusComboBox">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>55</y>
        <width>216</width>
        <height>20</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Open</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>&lt;all&gt;</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="shotgunLabel_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>116</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Set Shotgun filters:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="filterAssigneeComboBox">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>28</y>
        <width>216</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="shotgunLabel_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>56</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
     <widget class="QLabel" name="shotgunLabel_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>31</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Assignee:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="shotgunListWidget">
      <property name="geometry">
       <rect>
        <x>295</x>
        <y>50</y>
        <width>276</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="shotgunLabel">
      <property name="geometry">
       <rect>
        <x>295</x>
        <y>5</y>
        <width>256</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose a Shotgun Asset:</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="chrCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>120</width>
        <height>17</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="text">
       <string>Characters</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="prpCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>97</y>
        <width>120</width>
        <height>17</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="text">
       <string>Props</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="vehCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>114</y>
        <width>120</width>
        <height>17</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="text">
       <string>Vehicles</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="blgCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>120</width>
        <height>17</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="text">
       <string>Buildings</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="envCheckBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>86</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Environment</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="noneCheckBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>97</y>
        <width>86</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;null&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="filterLineEdit">
      <property name="geometry">
       <rect>
        <x>295</x>
        <y>25</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="mcCheckBox">
      <property name="geometry">
       <rect>
        <x>495</x>
        <y>25</y>
        <width>76</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Match case</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="shotgunCheckBox">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>175</y>
       <width>231</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Do not publish to Shotgun</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabNote">
    <attribute name="title">
     <string>Character Notes</string>
    </attribute>
    <widget class="QTextEdit" name="crTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>586</width>
       <height>526</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>595</y>
     <width>150</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <action name="actionShotgunSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="toolTip">
    <string>Activate all checkboxes.</string>
   </property>
  </action>
  <action name="actionShotgunSelectNone">
   <property name="text">
    <string>Select None</string>
   </property>
   <property name="toolTip">
    <string>Deactivate all checkboxes.</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>shotgunCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>frame</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
